recipes/python-gflags/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "2.0" %}
+
+package:
+  name: python-gflags
+  version: {{ version }}
+
+source:
+  fn: python-gflags-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/p/python-gflags/python-gflags-{{ version }}.tar.gz
+  md5: 23c9a793959a54971b1f094b0c6d03b1
+
+build:
+  number: 0
+  skip: True  # [py3k]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - gflags
+
+about:
+  home: https://github.com/google/python-gflags
+  license: BSD
```

Comment:
The linter should warn about that. I will open an issue and try to implement it.